rllib/algorithms/impala/impala.py

```diff
@@ -103,7 +105,6 @@ def __init__(self, algo_class=None):
         self.replay_buffer_num_slots = 0
         self.learner_queue_size = 16
         self.learner_queue_timeout = 300
-        self.max_requests_in_flight_per_sampler_worker = 2
```

Comment:
Dumb question: Where did this setting actually go? Do we use 2 hard-coded in the WorkerSet?
Should we make this configurable via `AlgorithmConfig` (or at least for all algos that do async sampling)?

Reply:
I feel like this is a very important parameter for tuning IMPALA/APPO/etc...

Reply:
it's now part of the base AlgorithmConfig.